shared/AppInsightsCommon/src/Util.ts

```diff
@@ -743,8 +743,8 @@ export class CorrelationIdHelper {
 }
 
 export class AjaxHelper {
-    public static ParseDependencyPath(logger: IDiagnosticLogger, absoluteUrl: string, method: string, commandName: string) {
-        let target, name = commandName, data = commandName;
+    public static ParseDependencyPath(logger: IDiagnosticLogger, absoluteUrl: string, method: string, commandName: string, customData: string) {
+        let target, name = commandName, data = customData || commandName;
```

Comment:
Update to use "data" value when "data" property exists, otherwise fallback to use commandName <-- the previous behavior.

Reply:
Looking back at the history of this field (takes quite a bit of digging), the change to have the "data" field being the sanitized version of the "name" is a requirement of downstream systems.

And by default the remote dependency doesn't populate the "data" field (I think because it's derived from name)